tasks/make-rootfs/run.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+SUFFIX="${STACKS_SUFFIX:-}"
+
+src-ci/concourse/tasks/common/start-docker.sh
```

Comment:
No, because [cf-suse-stacks](https://github.com/SUSE/cf-suse-stacks/blob/master/Makefile) runs `docker` all over the place :(

If _that_ gets changed to not do that, then yes.

The script was copied (fresh) from [the upstream job], just with a different name. And the fixed so it's not broken.

[the upstream job]: https://github.com/cloudfoundry/buildpacks-ci/blob/master/scripts/start-docker